test/spec-example/shmem_ctx.c

```diff
@@ -69,32 +59,29 @@ int main(void) {
         int ret = shmem_ctx_create(SHMEM_CTX_PRIVATE, &ctx);
 
         if (ret != 0) {
-            printf("%d: Warning, unable to create context (%d)\n", me, ret);
-            ctx = SHMEM_CTX_DEFAULT;
```

Comment:
We need to keep these changes so that the unit tests don't fail on implementations that can't create the context.